head-of-the-class/tests/gradeMocks.js

```diff
@@ -0,0 +1,64 @@
+const mockAssignmentAverages = [
```

Comment:
Not sure how this got in here. I suspect you had VS Code open to the folder above these projects instead of open at the specific folder you are working in. This is the type of problem that doing that causes. If you can see other folders in the VS Code side bar, quite and reopen in the actual folder.